.github/workflows/release-integration.yml

```diff
@@ -59,9 +59,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: docker/setup-qemu-action@v1
```

Comment:
let's bump `setup-qemu-action` to v2 too

Reply:
Done